mappings/net/minecraft/command/CommandDefaultGameMode.mapping

```diff
@@ -1,4 +1,11 @@
 CLASS bg net/minecraft/command/CommandDefaultGameMode
+	METHOD a setServerwideGamemode (Lalg;Lnet/minecraft/server/MinecraftServer;)V
```

Comment:
`setDefaultGamemode` is probably a better name here. Maybe `setDefaultGameMode`?